### PR TITLE
doc: Clarify multiple wallets for same signer (final)

### PR DESCRIPTION
**Pull request by Rspigler**
https://github.com/bitcoin/bitcoin/pull/25891

### CODE REVIEW

**Comment by Rspigler** [comment]
CI error https://github.com/bitcoin/bitcoin/issues/25644

File: doc/external-signer.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Create a wallet, this automatically imports the public keys:
 ```sh
 $ bitcoin-cli createwallet "hww" true true "" true true true
 ```
+Creating `N` distinctly named wallets with the same external signer, will create `N` identical wallets, with the same seed, account, and addresses - just named differently. New wallet creation will not automatically increment the `account` path.
```
Comment (luke-jr) [inline doc/external-signer.md:42]: Not identical, since metadata can(will?) differ

**Comment by Rspigler** [comment]
I'll squash later if I get ACKs

**Comment by achow101** [comment]
> I'll squash later if I get ACKs

Generally people only ACK things that are in a state where they could be merged, so it is preferable for PRs to be squashed (and rebased if needed) before seeking ACKs,

**Comment by MarcoFalke** [comment]
Maybe close for now and reopen when it is ready? This is a one-line doc change, so doesn't seem worth it to sit several months?

**Review by pablomartin4btc — COMMENTED** [review]
I guess this could be something unexpected from the point of view of a user, unless I'm missing something here, maybe should be specified as a warning? (e.g. "be aware of...")

**Review by hernanmarino — COMMENTED** [review]
Simple but nice addition to the documentation. @Rspigler any plans of squashing this so it can be merged ?

**Comment by fanquake** [comment]
Closing for now. @Rspigler please comment and this can be re-opened, after which you can squash the commits (pushing to the branch of a closed PR will break GitHub). @Sjors maybe you can also review?

**Pull request closed without merging**